commands/server.go

```diff
@@ -46,15 +46,16 @@ type serverCmd struct {
 	// Can be used to stop the server. Useful in tests
 	stop <-chan bool
 
-	disableLiveReload bool
-	navigateToChanged bool
-	renderToDisk      bool
-	serverAppend      bool
-	serverInterface   string
-	serverPort        int
-	liveReloadPort    int
-	serverWatch       bool
-	noHTTPCache       bool
+	disableLiveReload       bool
+	navigateToChanged       bool
+	renderToDisk            bool
+	renderStaticFilesToDisk bool
```

Comment:
I sugget renaming renderStaticFilesToDisk => renderStaticToDisk -- a little shorter and easier to type.